packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -2756,8 +2786,23 @@ abstract class Element implements BuildContext {
   Set<InheritedElement> _dependencies;
   bool _hadUnsatisfiedDependencies = false;
 
+  void _debugCheckStateIsActive() {
```

Comment:
this method name should explicitly call out that it's about looking up an ancestor, since that's what the message is about.